blog/index.html

```diff
@@ -0,0 +1,112 @@
+        <img src="/img/logo-knicka-wide.png" alt="knicka office" />
+      </h1>
+      <nav class="global-nav">
+        <label for="GLOBAL_NAV_TOGGLE_BUTTON" class="global-nav__toggle-button">
+          <div class="global-nav__button-line parts-toggle-munu">
+            <span></span><span></span><span></span>
+          </div>
+        </label>
+        <input
+          type="checkbox"
+          id="GLOBAL_NAV_TOGGLE_BUTTON"
+          class="common-display-toggle-check"
+        />
+        <ul class="global-nav__menu common-nav-list">
+          <li class="global-nav__item common-nav-list__item">
+            <a href=""><span>ABOUT</span><span>オフィスのこと</span></a>
+          </li>
+          <li class="global-nav__item common-nav-list__item">
+            <a href=""><span>CLIENT</span><span>顧客</span></a>
+          </li>
+          <li class="global-nav__item common-nav-list__item">
+            <a href=""><span>BLOG</span><span>ブログ</span></a>
+          </li>
+          <li class="global-nav__item common-nav-list__item">
+            <a href=""><span>CONTACT</span><span>お問い合わせ</span></a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="main-content-wrapper">
+    <article class="blog-main">
+        <div class="blogHeaderNav">
+            <button class="blogHeaderNav__day-before">
+              <p>前日</p>
+            </button>
+        </div>
+    <h2>令和元年12月20日に考えたこと</h2>
+    <div class="blog-main__content-height">    
+    <div class="blog-main__content">
+      <h3>理屈に頼り過ぎず感覚も大事にする</h3>
+
+      <p>ニッカオフィスのウチヤマケンイチです。</p>
+
+      <p>物事を考えるとき、あわせてその理由もセットで考える癖が僕にはあります。</p>
+      <p>思いつきの意見じゃなくて、きちんとロジカルに話を組み立てることは仕事ではとても重要で、理由とセットで思考する癖。訓練して身につけたわけです。</p>
+
+      <p>ただ最近、そんなふうに理由に囚われてばかりいると、思考が狭まってしまうというか、なんとも窮屈な感じを覚えるようになりました。</p>
+      <p>直感で思いついた案があっても、理由がつけられないからとその案を捨てる。本当にこれで良いのでしょうか。</p>
+
+      <p>例えば、飲食店を始めたいとか、日本一周したいとか。そんなふうになんとなく思いついたことはあっても、ではそれをやる合理的な理由は？と聞かれると、答えられなかったり。</p>
+      <p>であればそれらはやるべきでないのかというと、それはそれで違う気もするのです。</p>
+
+      <p>プライベートはもちろん、仕事についてもある程度は感覚で進めてしまうことも必要なのかもしれない。</p>
+      <p>ロジカルと感覚のどっちが正しいというよりも、ロジカルには説明できないことも結構あるんじゃないかということ。いや説明しようとすればできるのかもしれないけれど、時間がかかってしまうし、その間に鮮度も落ちてしまうのかなと、そう思うのです。</p>
+
+      <p>思考の鮮度。生きている考えというのは、一本釣りのように突然現れるものなのかもしれません。</p>
+      <p>理屈に頼り過ぎず感覚も大事にする。きっとそういうターンもあるものさ。</p>
+
+      <p>ウチヤマ ケンイチ</p>
+    </div>
+    </div>
+      <div class="blog-main__card">
+        <img class="blog-main__card" src="./img/blog-one-point.png" alt="blog-one-point"></img>
+      </div>
+    </article>
+
+    <div class="blogFooterNav">
+      <button class="blogFooterNav__day-before">
+        <p class="blogFooterNav__day-before-zenjitsu">前日</p>
+        <p class="blogFooterNav__day-before-title">ニッカオフィスのウチヤマケンイチです。きちんとしたレッスンのおかげでゴル
+      </button>
+      <button class="blogFooterNav__day-after">
+        <p class="blogFooterNav__day-after-title">「普通の人」なんていない</p>
+        <p class="blogFooterNav__day-after-yokujitsu">翌日</p>

```

Comment:
```suggestion
    <div class="blog-footer-nav">
      <button class="blog-footer-nav__day-before">
        <p class="blog-footer-nav__day-before-zenjitsu">前日</p>
        <p class="blog-footer-nav__day-before-title">ニッカオフィスのウチヤマケンイチです。きちんとしたレッスンのおかげでゴル
      </button>
      <button class="blog-footer-nav__day-after">
        <p class="blog-footer-nav__day-after-title">「普通の人」なんていない</p>
        <p class="blog-footer-nav__day-after-yokujitsu">翌日</p>

```
class名は、キャメルケースでは無く、ハイフンケースが良いです。